spec/fixtures/searchworks_pub_date_data.rb

```diff
@@ -782,17 +782,17 @@
       mods_origin_info_start_str +
         '<dateCreated encoding="edtf" point="start" keyDate="yes">-18</dateCreated>' +
         '<dateCreated encoding="edtf" point="end">-17</dateCreated>' +
-        mods_origin_info_end_str => [-18, '18 B.C. - 17 B.C.'],
+        mods_origin_info_end_str => [-18, '19 B.C. - 17 B.C.'],
```

Comment:
actually, the second date should be changed too, for when we really do date ranges.  However, that display string is going to be via imprint_display code anyway, so if it matters, I'll get it in the coding changes I'm working on now.